config/packages/doctrine.yaml

```diff
@@ -10,6 +11,7 @@ doctrine:
         enable_lazy_ghost_objects: true
         naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
         auto_mapping: true
+        report_fields_where_declared: false
```

Comment:
you should try to make it `true` to opt-in for the new behavior (keeping it as `false` to keep the old validation behavior is what is deprecated)